src/main/kotlin/org/rust/ide/intentions/RunCargoExpandIntention.kt

```diff
@@ -61,5 +70,26 @@ class RunCargoExpandIntention : RsElementBaseIntentionAction<RunCargoExpandInten
 
     companion object {
         private const val PATH_SEPARATOR: String = "::"
+
+        private fun checkIsNightlyToolchain(project: Project, cargoTarget: CargoWorkspace.Target): Boolean {
+            val channel = project.computeWithCancelableProgress("Fetching rustc version...") {
+                project.toolchain?.rustc()?.queryVersion(cargoTarget.pkg.rootDirectory)?.channel
+            }
+            if (channel == RustChannel.NIGHTLY) return true
```

Comment:
Note, you shouldn't **use** nightly compiler. It's enough to **have** installed nightly toolchain.
Also, what's about `dev` channel?